docs/rules/no-extraneous-dependencies.md

```diff
@@ -19,6 +19,19 @@ You can set the options like this:
 "import/no-extraneous-dependencies": ["error", {"devDependencies": false, "optionalDependencies": false, "peerDependencies": false}]
 ```
 
+You can also use globs instead of literal booleans:
+
+```js
+"import/no-extraneous-dependencies": ["error", {"devDependencies": "*.test.js"}]
```

Comment:
it seems like it'd be simpler to just require this to always be an array - eslint can enforce it, and that way the user would get an immediate indication that multiple globs were possible.